src/GloryDuckTables2019Lkl.cc

```diff
@@ -250,28 +255,27 @@ Int_t GloryDuckTables2019Lkl::ReadGloryDuckInputData(TString filename)
       getline(ff,LklLine);
 
       // read rest of the line
-      vlkl2D[col] = vector<Double_t>(mass.size());
-      row = 0;
+      vlkl2D[row] = vector<Double_t>(mass.size());
+      col = 0;
       std::istringstream line(LklLine);
       while (line >> readingLkl)
         {
-          vlkl2D[col][row] = readingLkl;
-          row++;
+          vlkl2D[row][col] = readingLkl;
+          col++;
         }
-      // check if line was complete or not HERE -1 LATER
-      if(vlkl2D[col].size() != mass.size())
+      if(col != mass.size())
         {
-          cout << "You have a different line size length for line " << col+1 << " ! Number of mass values = " << mass.size() << " while number of likelihood values = " << vlkl2D[col].size() << endl;
+          cout << "You have a different line size length for line " << row+1 << " ! Number of mass values = " << mass.size() << " while number of likelihood values = " << col << endl;
```

Comment:
Here you nedd "row+2" instead of "row+1" because of the first line that contains logJ and masses.

Reply:
Okay, I will change this! I started counting from 0 ;)